crates/chia-sdk-test/src/simulator.rs

```diff
@@ -217,6 +247,64 @@ impl Simulator {
                 .copied()
                 .unwrap_or(CoinState::new(coin, None, Some(self.height)));
 
+            if let Some(relative_height) = spend.height_relative {
+                let Some(created_height) = coin_state.created_height else {
+                    return Err(SimulatorError::Validation(
+                        ErrorCode::EphemeralRelativeCondition,
```

Comment:
these error codes don't look right. I would expect this error code to mean "an ephemeral spend emitted a relative time/height condition" (which I believe isn't allowed)

Reply:
I'll take a look at this later, since it's just in the simulator